src/caffe/layers/crop_layer.cu

```diff
@@ -39,10 +39,10 @@ void CropLayer<Dtype>::crop_copy_gpu(const vector<Blob<Dtype>*>& bottom,
                 src_data, dest_data, is_forward);
     }
   } else {
-    // We are at the last two dimensions, which are stored continuously in memory
-    // With (N,C,H,W)
-    //      (0,1,2,3) cur_dim   -> H
-    //                cur_dim+1 -> W
+    // We are at the last two dimensions, which are stored continuously in
+    // memory With (N,C,H,W)
```

Comment:
Thanks for the fix @CDLuminate.  Sorry for the nitpick but could you change this to `memory.` (add period `.`) so that the capitalized `With` immediately following doesn't look odd?